staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd_test.go

```diff
@@ -160,6 +163,64 @@ func TestDelete(t *testing.T) {
 	test.TestDelete(validNewCustomResource())
 }
 
+func TestGeneration(t *testing.T) {
+	// enable alpha feature CustomResourceSubresources
+	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.CustomResourceSubresources, true)()
+
+	storage, server := newStorage(t)
+	defer server.Terminate(t)
+	defer storage.CustomResource.Store.DestroyFunc()
+	modifiedRno := *validNewCustomResource()
+	modifiedRno.SetGeneration(10)
+	setObservedGeneration(&modifiedRno, 10)
```

Comment:
observedGeneration doesn't really have any meaning for custom resources. It serves no other purpose in this test other than to check that status is not set during initialization. This is already being checked in other tests.

Moreover, this test should focus on `generation`. So I think we don't need to include `observedGeneration` in this test.

Reply:
Done